src/app/components/register/register.component.ts

```diff
@@ -1,53 +1,71 @@
 import { RegisterRequest } from './../../register-request';
 import { CommonModule } from '@angular/common';
-import { Component } from '@angular/core';
-import {FormBuilder, FormControl, FormGroup, ReactiveFormsModule, Validators} from '@angular/forms'
+import { Component, inject } from '@angular/core';
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
 import { AuthService } from '../../services/auth.service';
 import { Router } from '@angular/router';
+import { ToastrService } from 'ngx-toastr';
 
 @Component({
   selector: 'app-register',
   standalone: true,
   imports: [ReactiveFormsModule, CommonModule],
   templateUrl: './register.component.html',
-  styleUrl: './register.component.scss'
+  styleUrl: './register.component.scss',
 })
 export class RegisterComponent {
+  toaster = inject(ToastrService);
+
+  email: FormControl<string | null> = new FormControl<string>('', [
+    Validators.required,
+    Validators.email,
+  ]);
+  password: FormControl<string | null> = new FormControl<string>('', [
+    Validators.required,
+    Validators.minLength(5),
+  ]);
 
-  email : FormControl<string | null> = new FormControl<string>('', [Validators.required, Validators.email]) 
-  password : FormControl<string | null> = new FormControl<string>('', [Validators.required, Validators.minLength(5)]) 
-  
   registerForm!: FormGroup;
 
-  inlineNotification: {show: boolean; type: string; text: string} = {
+  inlineNotification: { show: boolean; type: string; text: string } = {
     show: false,
     type: '',
-    text: ''
-  }
+    text: '',
+  };
```

Comment:
This is declared in separate file, it will be good to have a single service that get's called to handle this

Reply:
service has been created to handle that